src/include/simeng/arch/aarch64/helpers/sve.hh

```diff
@@ -784,17 +790,19 @@ class sveHelp {
   }
 
   /** Helper function for SVE instructions with the format `inc<b, d, h, w>
-   * xdn{, pattern{, #imm}}`.
+   * xdn{, pattern{, MUL #imm}}`.
    * T represents the type of operation (e.g. for INCB, T = int8_t).
    * Returns single value of type uint64_t. */
```

Comment:
Should this say int64_t (resp. uint64_t) now?

Reply:
yes it should